setup.cfg

```diff
@@ -71,6 +72,7 @@ universal = 1
 test=pytest
 
 [tool:pytest]
+DJANGO_SETTINGS_MODULE=tests.settings
```

Comment:
Why was it necessary to move the settings from [`pytest_configure`](https://pytest-django.readthedocs.io/en/latest/configuring_django.html#using-django-conf-settings-configure) to [`tests/settings.py`](https://pytest-django.readthedocs.io/en/latest/configuring_django.html#the-environment-variable-django-settings-module)? Only to create migration modules or something else?

Reply:
The Django 2.2 environments fail due to an unknown problem wrt running the migrations within pytest-django.

So creating and running the migrations manually using `django-admin` requires a normal `settings.py` file.

Reply:
Thanks for the info :+1: